packages/frontend/src/features/scheduler/components/SchedulerDeleteModal.tsx

```diff
@@ -1,26 +1,23 @@
-import {
-    Button,
-    Dialog,
-    DialogBody,
-    DialogFooter,
-    DialogProps,
-    NonIdealState,
-    Spinner,
-} from '@blueprintjs/core';
+import { Box, Button, Group, Loader, Modal, Stack, Text } from '@mantine/core';
+import { IconTrash } from '@tabler/icons-react';
 import React, { FC, useCallback, useEffect } from 'react';
 import ErrorState from '../../../components/common/ErrorState';
+import MantineIcon from '../../../components/common/MantineIcon';
 import { useScheduler } from '../hooks/useScheduler';
 import { useSchedulersDeleteMutation } from '../hooks/useSchedulersDeleteMutation';
 
-interface DashboardDeleteModalProps extends DialogProps {
+interface DashboardDeleteModalProps {
     schedulerUuid: string;
     onConfirm: () => void;
+    onClose: () => void;
+    isOpen: boolean;
```

Comment:
not sure if we want to pick from `ModalProps` from `Mantine` here. What do you think?

Reply:
Sure, I can do that